app/src/main/java/com/github/sdp/ratemyepfl/activity/classrooms/RoomReviewsListActivity.kt

```diff
@@ -44,27 +45,17 @@ class RoomReviewsListActivity : AppCompatActivity() {
 
     }
 
-
     /* Adds review */
     private fun fabOnClick() {
-        val intent = Intent(this, AddRoomReviewActivity::class.java)
-        intent.putExtra(AddRoomReviewActivity.EXTRA_ROOM_REVIEWED, viewModel.id)
+        val intent = Intent(this, AddReviewActivity::class.java)
+        intent.putExtra(AddReviewActivity.EXTRA_ITEM_REVIEWED, viewModel.id)
         resultLauncher.launch(intent)
     }
 
     private var resultLauncher =
         registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
             if (result.resultCode == Activity.RESULT_OK) {
-                val d: Intent? = result.data
-                val roomGrade: Int? = d?.getIntExtra(ROOM_GRADE, INVALID_ROOM_GRADE)
-                val roomComment = d?.getStringExtra(ROOM_COMMENT)
-                if (roomGrade != null && roomGrade != INVALID_ROOM_GRADE && roomComment != null) {
-                    viewModel.insertReview(roomGrade, roomComment)
-                }
+                // refresh viewModel here
```

Comment:
The refresh could be a swipe (e.g., https://developer.android.com/training/swipe). Task for the next sprint I guess.

Reply:
Julien is going to do it this sprint I think

Reply:
Absolutely